spec/views/spree/admin/shared/_order_links.html.haml_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+
+describe "spree/admin/shared/_order_links.html.haml" do
+  helper Spree::BaseHelper # required to make pretty_time work
+
+  around do |example|
+    original_config = Spree::Config[:enable_invoices?]
+    example.run
+    Spree::Config[:enable_invoices?] = original_config
```

Comment:
were you trying to set `enable_invoices?` to true? I guess so but this block is not changing anything, just reassigning its current value.

Reply:
I noticed this is not necessary